Base/usr/share/man/man1/gunzip.md

```diff
@@ -1,22 +0,0 @@
-## Name
-
-gunzip
```

Comment:
Do we want `zcat` and `gunzip` manpages to be symlinks to `gzip` (and maybe merging the synopses (?) into there) instead of fully removing them? That is, of course, only if symlinks won't break any of our tooling.

Reply:
Sure. I added symlinks. Hopefully this will not break anything.